### ISSUE TITLE
Out of range months

### ISSUE THREAD

**Issue by danielzlotnik** [airbnb/react-dates#339]
Would be great if we could disable navigation to "out of range" months (defined by a function, similar to out of range days).

Months where all days "out of range" should be out of range by default (and therefore unnavigatable) unless explicitly allowed.

Thanks!

**Comment by ljharb** [comment]
If you're in April, and all of March is out of range, but February is in range, what happens when you try to go back a month?

**Comment by danielzlotnik** [comment]
@ljharb rephrased the issue.

**Comment by ljharb** [comment]
min/max date props might indeed make sense, thanks,

**Comment by epifab** [comment]
min and max is for some cases a must have.. any progress on this?

**Comment by ljharb** [comment]
Nobody’s submitted a PR yet.

**Comment by erin-doyle** [comment]
This is on my team's list of things we'd like to work on at some point but it's pretty low in priority so if someone else needs it faster I'd suggest submitting a PR.  Otherwise I can get it it eventually. :woman_shrugging:

**Comment by majapw** [comment]
Same here --- it is on the list, but it is not yet a high priority. The fastest way to get to it would be to submit a PR! :)

**Comment by danielzlotnik** [comment]
@fernandofleury you'd still be able to navigate through all those unavailable months.

**Comment by fernandofleury** [comment]
@danielzlotnik that is true. but personally, I don't see that as a big UI concern. but it might be different for your case

**Comment by Vgurnani** [comment]
> Would be great to have the ability to limit months navigation to a specific range.
> 
> Could be achieved by min / max date props.
> 
> Thanks!

I tried with adding minDate in my wrapper but its now working as expected.
